test/storage-luatest/storage_1_1_test.lua

```diff
@@ -157,3 +157,99 @@ test_group.test_bucket_send_field_types = function(g)
         box.space.test:drop()
     end)
 end
+
+--
+-- Custom triggers on bucket changes
+--
+test_group.test_on_bucket_event = function(g)
+    -- Initial problem: buckets migration looks like a regular
+    -- client data activity, indistinguishable in the replication flow.
+    -- In order to allow efficient change data capturing we can add a fake
+    -- operation in the beginning of the transaction, so that a client could
+    -- determing which operations can be skipped (using HEADER.tsn) as they
+    -- were done not by the user but by gc or rebalancer.
+    vtest.cluster_exec_each_master(g, function(engine)
+        -- Space, in which these fake operations will be written
+        rawset(_G, 'notify_name', '_vshard_migration_notify_' .. engine)
```

Comment:
Lets make the name less conflicting with potential future vshard-private naming and make it something like `test_migration_notify` or similar.